src/Components/SavedSearchAlert/Components/FrequencyRadioButtons.tsx

```diff
@@ -0,0 +1,50 @@
+import { BorderedRadio, Box, RadioGroup, Text } from "@artsy/palette"
+import { themeGet } from "@styled-system/theme-get"
+import styled from "styled-components"
+import { useSystemContext } from "System"
+import { isArtsyEmail } from "Utils/isArtsyEmail"
```

Comment:
**suggestion**: I know this Email-based utility is already in use but I think we should prefer the `userIsTeam` util to determine whether the currently authenticated user works at Artsy. Seems less brittle than Email-based detection. What do you think?

Reply:
Yes, I agree with you. This approach would be more correct to use, since any user can potentially sign up with an email that includes `@artsymail.com` and use the account without confirmation

https://user-images.githubusercontent.com/3513494/186893848-418df42c-3c1d-4e77-ad6b-3af95f2e72f4.mp4

Reply:
but I think we should also use `userIsAdmin`

Reply:
Ahh, I think "internal only" would work with the team role.

Reply:
To clarify, if we change the copy to be "internal only", I think that would align with the `team` role check. All provisioned Artsy employee accounts have the `team`, user role.

Reply:
did I understand you correctly:
1. here change copy from `Instant (beta / admin only)` to `Instant (beta / internal only)`
2. here we leave [the check](https://github.com/artsy/force/blob/71e3079e0620c7f9a13a5ff90498cc073ab6b28e/src/Components/SavedSearchAlert/Components/FrequencyRadioButtons.tsx#L18) only for `userIsTeam`

is that right?

Reply:
Yes, perfect!

Reply:
Done
<img width="446" alt="image" src="https://user-images.githubusercontent.com/3513494/186919001-3c90ee17-4a4a-44ff-9cb3-21e2e8962077.png">